doc/General/Contributing.md

```diff
@@ -55,21 +55,26 @@ Copyright
 We recommend that if you add a new file containing original code to the code
 base that you include a copyright notice in it as per the Free Software
 Foundation's guidelines.  You might find something like the following header
-appropriate (although this is not legal advice ;-):
+appropriate (although this is not legal advice ;-). Please also ensure you add 
+the package information to the header.
 ```
-  <?php
-  /*
-   * LibreNMS module to frob blurgs from a foo bar
-   *
-   * Copyright (c) 2015 Internet Widgitz Pty Ltd <http://example.com/>
-   *
-   * This program is free software: you can redistribute it and/or modify it
-   * under the terms of the GNU General Public License as published by the
-   * Free Software Foundation, either version 3 of the License, or (at your
-   * option) any later version.  Please see LICENSE.txt at the top level of
-   * the source code distribution for details.
-   */
-  ?>
+<?php
+/*
+ * Copyright (c) 2015 Internet Widgitz Pty Ltd <http://example.com/>
```

Comment:
I don't add this on mine as it seems redundant to the one below.  I'm fine with it though.